src/NuGet.Clients/NuGetConsole.Host.PowerShell/Scripts/NuGet.PackageManagement.PowerShellCmdlets.dll-Help.xml

```diff
@@ -2,6 +2,7 @@
 
 <helpItems schema="maml">
 
+  <!-- Please note below command names are hard coded in emitting telemetry in VsPowerShellHostTelemetryEmit.cs. If you add/remove command here then please update it there too. -->
```

Comment:
The help seems like the wrong spot here. 

Commands are added in the psm1, I'd recommend adding it there.

Reply:
Ok. I'll move.

Reply:
Moved nuget.ps1 file.